pkgs/desktops/kde-5/applications/akonadi-contacts.nix

```diff
@@ -0,0 +1,41 @@
+{ akonadi
+, akonadi-mime
+, boost
+, ecm
+, grantlee
+, kdeApp
+, kcalcore
+, kcontacts
+, kdelibs4support
+, kdoctools
+, kio
+, kmime
+, lib
+, libical
+, qt5
```

Comment:
Use `qtwebengine` directly to ensure that everything gets built with the same version of Qt 5.